src/versionbits.cpp

```diff
@@ -15,115 +15,115 @@ struct ForkDeploymentInfo VersionBitsDeploymentInfo[Consensus::MAX_VERSION_BITS_
     },
     {
         /*.name =*/(char *)"", // unallocated bit 1
-        /*.gbt_force =*/false,
+        /*.gbt_force =*/true,
     },
```

Comment:
why change these now to true?

Reply:
These bits are unused so could be true or false.  However, if some bug caused an unused bit to be considered, a "false" value may have undesired effects like preventing a block template from being constructed.

Reply:
false is the more active value.  It allows the client to turn the bit off in STARTED state and forces the client to include the fork in his rules array in ACTIVE state.  true prevents him from turning it off in STARTED state and doesn't force him to include it in his rules array.